packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -119,14 +125,21 @@ export const getNotificationPageData = async (
     let csvUrl;
     let csvUrls;
     let pdfFile;
+
+    const schedulerUuid =
+        'schedulerUuid' in scheduler &&
+        typeof scheduler.schedulerUuid === 'string'
+            ? scheduler.schedulerUuid
+            : undefined;
```

Comment:
added all these checks so it conforms to the backwards compatibility mentioned in: https://github.com/lightdash/lightdash/blob/chore/allow-for-minimal-to-have-access-to-scheduler-uuid/packages/backend/src/scheduler/SchedulerTask.ts#L295
But the `schedulerUuid` value is always set on new and recent scheduled deliveries